Source/Blazorise/Utilities/Parsers.cs

```diff
@@ -153,6 +153,12 @@ public static bool TryParseDate<TValue>( string value, DateInputMode inputMode,
                 return true;
             }
 
+            if ( type == typeof( DateOnly ) && DateOnly.TryParse( value, out var parsedDto3 ) )
+            {
+                result = (TValue)(object)parsedDto3;
```

Comment:
These seem hella weird. Would this be better or the same?? 
It will still be a boxing operation I guess. Just don't know if more efficient?
![Box](https://user-images.githubusercontent.com/22283161/153716628-56f57944-1f15-4c53-8374-db3e504504b3.png)

Reply:
We could use boxing but since we have at this point already parsed the value it is safe to convert it to a TValue.

Reply:
I didn't understand. So you're saying you prefer the solution I posted?

Reply:
Aaah I was focused on the screenshot and failed to see that we already use boxing 😂, Yeah, your solution seems better.

Reply:
Or maybe not

![image](https://user-images.githubusercontent.com/900302/153717214-6fab1140-028e-4411-85ef-6326bba2fd67.png)

Reply:
What do you mean? In your screenshot I think you are missing the where constraint, that's why it's complaining.